tasks.py

```diff
@@ -34,6 +34,19 @@ def clean(bower=False, node=False):
         info('Removing {0}'.format(pattern))
         run('cd {0} && rm -rf {1}'.format(ROOT, pattern))
 
+@task
+def update(data=False):
```

Comment:
Because it migrate data. Loading a new database can be very long so I prefer the dev to be conscious that it will be migrating its data too by explicitely adding `-d` or `--data` on the command line. What's your opinion ?